lib/scope.js

```diff
@@ -144,7 +145,7 @@ ScopeDefinition.prototype.related = function(receiver, scopeParams, condOrRefres
           queryRelated.where[IdKey] = collectTargetIds(data, IdKey);
         }
 
-        relatedModel.find(queryRelated, cb);
+        relatedModel.find(queryRelated, options, cb);
```

Comment:
Thank you dmellonch 
There is a similar PR created for solving same issue, details please check 
https://github.com/strongloop/loopback-datasource-juggler/pull/1255#issuecomment-290157667

Briefly, I feel the `options` should be applied to either `relatedModel.find` or `targetModel.find` but **NOT** both, I prefer the changes made in this PR, just we should remove the `options` in [Line 109](https://github.com/strongloop/loopback-datasource-juggler/pull/1426/files#diff-c2a679b29102d5574291e884531cb8aeR109)